boa_engine/src/builtins/intl/mod.rs

```diff
@@ -29,13 +33,22 @@ impl BuiltIn for Intl {
         let _timer = Profiler::global().start_event(Self::NAME, "init");
 
         let string_tag = WellKnownSymbols::to_string_tag();
+        let date_time_format = ConstructorBuilder::new(context, DateTimeFormat::constructor)
```

Comment:
```suggestion
        let date_time_format = DateTimeFormat::init(context);
```
Here we can just call the init static method.

Reply:
Done https://github.com/boa-dev/boa/pull/2025/files#diff-1dd4af22d3a6142a8060d056e9d0489d84d8a4d9c68a8f6e7b359b8d3ab35499R36